### ISSUE TITLE
Lerna with hoisted modules - mismatched node_modules resolution

### ISSUE THREAD

**Issue by chrisblossom** [wallabyjs/public#1442]
### Issue description or question
Continuing issue https://github.com/wallabyjs/public/issues/1402.

It seems the following node_modules resolution is not working as expected.

```js
    process.env.NODE_PATH = require('path').join(
        wallaby.localProjectDir,
        '../../node_modules',
    );
```

The issue is found with the latest Gitlab commit (specifically happened with 98b116d4ee46167931ce1b47f880c5612199dd44) I gave you access to in the previous issue.

```
TypeError: Cannot read property 'close' of undefined
    at Object.<anonymous> (/Users/chris/github/app/node_modules/chalk/index.js:72:75)
    at Runtime._execModule (/Users/chris/github/app/node_modules/jest-runtime/build/index.js:541:13)
    at Runtime.requireModule (/Users/chris/github/app/node_modules/jest-runtime/build/index.js:330:14)
    at Runtime.requireInternalModule (/Users/chris/github/app/node_modules/jest-runtime/build/index.js:339:17)
    at options.isInternalModule.moduleName (/Users/chris/github/app/node_modules/jest-runtime/build/index.js:681:24)
    at Object.<anonymous> (/Users/chris/github/app/node_modules/jest-matcher-utils/build/index.js:10:14)
    at Runtime._execModule (/Users/chris/github/app/node_modules/jest-runtime/build/index.js:541:13)
    at Runtime.requireModule (/Users/chris/github/app/node_modules/jest-runtime/build/index.js:330:14)
```

Works as expected if I install without lerna and remove the node resolution. (as well as remove local lerna-deps)

Let me know if you need anymore information from me!

Thank you!

### Code editor or IDE name and version

```
WebStorm 2017.3.1
Build #WS-173.3942.31, built on December 11, 2017
JRE: 1.8.0_152-release-1024-b8 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
```

### OS name and version

``Mac OS X 10.13.2``

**Comment by ArtemGovorov** [comment]
Thanks for submitting the issue. It's fixed and the fix is published in the [latest core v1.0.532](https://wallabyjs.com/docs/intro/troubleshooting.html#forcing-wallabyjs-core-update).

**Comment by chrisblossom** [comment]
Verified it is working now. Thank you!!
